src/pages/docs/test-management/settings/notifications/overview.md

```
@@ -0,0 +1,82 @@
+Test Management by Testsigma allows you to integrate with Slack and Microsoft Teams and configure custom notifications for various test entities. Notifications help teams stay aligned with ongoing testing activities by alerting users about key events such as assignments, status changes, and review updates. This article discusses setting up notifications in Test Management by Testsigma.
+
+---
+
+> ## **Prerequisites**
+> 
+> Before you begin, ensure that you have an active account in the tools where you want to receive notifications.
+
+---
+
+## **Tools Available for Notification Integration**
+   - [Email](https://testsigma.com/docs/test-management/settings/notifications/email/)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)